src/index.html

```diff
@@ -1,13 +1,138 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>The MET</title>
+    <link
+    type="image/png"
+    href="images/logo.png"
+    rel="shortcut icon"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Manrope:wght@400;700&family=Playfair+Display:ital,wght@0,400;0,700;1,400;1,700&family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body" id="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a href="#page__body" class="top-bar__logo">
+          <img src="images/logo.png" alt="The Met logo">
+        </a>
+        <!-- розглядала варіант створити блок який буде мати
```

Comment:
it will be the preferable case since that should be real links
ask in the chat if you need some help with the implementation (you should use combined (chained) selector)

Reply:
P.S. Just for the future - it's better to ask such questions in the chat - the response time will be much shorter.